### PR TITLE
Provide translation hints for add-on admin links

### LINKED ISSUE #6607: Words in strings unclear as a verb or noun
### Describe the problem and steps to reproduce it:

Some of the words in the strings can be translated as verb or adjective. Need note or clarification on the use of these words
### What happened?
https://pontoon.mozilla.org/sl/amo-frontend/all-resources/?search=add-on&string=191281
- Admin add-on
- Admin add-on status
- Content review add-on

### What did you expect to happen?
Have comments to indicate if the word preceded add-on is used as a verb or adjective?

add-on for admin?
add-on for content review?

Or as a verb to 

administrate add-on and review add-on (what's the difference between content review add-on and review add-on).

### PR DESCRIPTION
**Pull request by bobsilverberg**
Fixes #6607 

@wagnerand and/or @jvillalobos Could one of you please review this? I made a best guess at what each of these links are for, but I could very well be wrong about some or all of them.

### CODE REVIEW

File: src/amo/components/AddonAdminLinks/index.js [hunk 1]
```diff
@@ -106,14 +109,17 @@ export class AddonAdminLinksBase extends React.Component<InternalProps> {
           className="AddonAdminLinks-contentReview-link"
           href={`/reviewers/review-content/${addon.slug}`}
         >
-          {i18n.gettext('Content review add-on')}
+          {// translators: This action allows a user to perform a content review of an add-on.
```
Comment (kumar303) [inline src/amo/components/AddonAdminLinks/index.js]: Does this work? I'm not sure if it does.

```js
{/* translators: This action allows a user to perform a content review of an add-on. */}
{i18n.gettext('Content review add-on')}
```

Comment (bobsilverberg) [reply src/amo/components/AddonAdminLinks/index.js]: I tested your syntax and it doesn't seem to work for extraction.

File: src/amo/components/AddonAdminLinks/index.js [hunk 2]
```diff
@@ -72,7 +72,8 @@ export class AddonAdminLinksBase extends React.Component<InternalProps> {
             addon.type === ADDON_TYPE_THEME ? 'theme' : 'addon'
           }/${addon.slug}/edit`}
         >
-          {i18n.gettext('Edit add-on')}
+          {// translators: This action allows a user to edit an add-on's properties.
```
Comment (wagnerand) [inline src/amo/components/AddonAdminLinks/index.js]: `s/a user/the add-on developer or an admin/`

File: src/amo/components/AddonAdminLinks/index.js [hunk 3]
```diff
@@ -83,7 +84,8 @@ export class AddonAdminLinksBase extends React.Component<InternalProps> {
           className="AddonAdminLinks-admin-status-link"
           href={`/admin/addon/manage/${addon.slug}/`}
         >
-          {i18n.gettext('Admin add-on status')}
+          {// translators: This action allows a user to edit the status of an add-on.
```
Comment (wagnerand) [inline src/amo/components/AddonAdminLinks/index.js]: `s/a user/an admin/`

File: src/amo/components/AddonAdminLinks/index.js [hunk 4]
```diff
@@ -95,7 +97,8 @@ export class AddonAdminLinksBase extends React.Component<InternalProps> {
             className="AddonAdminLinks-admin-link"
             href={`/admin/models/addons/addon/${addon.id}`}
           >
-            {i18n.gettext('Admin add-on')}
+            {// translators: This action allows a user to maintain an add-on.
```
Comment (wagnerand) [inline src/amo/components/AddonAdminLinks/index.js]: `s/a user/an admin/`

File: src/amo/components/AddonAdminLinks/index.js (hunk 1 above)
Comment (wagnerand) [inline src/amo/components/AddonAdminLinks/index.js]: `s/a user/a reviewer/`

File: src/amo/components/AddonAdminLinks/index.js [hunk 5]
```diff
@@ -106,14 +109,17 @@ export class AddonAdminLinksBase extends React.Component<InternalProps> {
...
+          {// translators: This action allows a user to perform a content review of an add-on.
+          i18n.gettext('Content review add-on')}
         </a>
       </li>
     ) : null;
 
     const codeReviewLinkText = isTheme(addon.type)
-      ? i18n.gettext('Review theme')
-      : i18n.gettext('Review add-on code');
+      ? // translators: This action allows a user to perform a review of a theme.
```
Comment (wagnerand) [inline src/amo/components/AddonAdminLinks/index.js]: `s/a user/a reviewer/`

File: src/amo/components/AddonAdminLinks/index.js [hunk 6]
```diff
@@ -106,14 +109,17 @@ export class AddonAdminLinksBase extends React.Component<InternalProps> {
...
+        i18n.gettext('Review theme')
+      : // translators: This action allows a user to perform a review of an add-on's code.
```
Comment (wagnerand) [inline src/amo/components/AddonAdminLinks/index.js]: `s/a user/a reviewer/`

**Comment by bobsilverberg** [comment]
Thanks for the review @wagnerand. I've updated the hints if you want to take another look.

**Pull request merged**